x-pack/plugins/security_solution/public/common/components/visualization_actions/configs/common/external_alert.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { GetLensAttributes, LensAttributes } from '../../types';
+
+export const getExternalAlertConfigs: GetLensAttributes = (stackByField = 'event.module') => {
+  return {
+    title: 'External alerts',
+    description: '',
+    visualizationType: 'lnsXY',
+    state: {
+      visualization: {
+        title: 'Empty XY chart',
+        legend: {
+          isVisible: true,
+          position: 'right',
+        },
+        valueLabels: 'hide',
+        preferredSeriesType: 'bar_stacked',
+        layers: [
+          {
+            layerId: 'a3c54471-615f-4ff9-9fda-69b5b2ea3eef',
```

Comment:
why are these ids like this? hard coded ids make me nervous. if they're truly supposed to be hard coded, could we use readable words instead of an id?